src/components/DataDecorator/Decorator/Decorator.js

```diff
@@ -86,9 +91,13 @@ class Decorator extends Component {
       );
     }
 
-    if (onClick) {
+    if (onClick || onContextMenu) {
```

Comment:
To me it should only render as a button if `onClick` is provided, otherwise someone could provide just `onContextMenu` and render a `Decorator` that as an interactive element (a button) but cannot actually be left-clicked or activated with the `space` or `enter` keys. That seems like it would cause problems for folks using assistive tech?

Reply:
I will look at this! My use case only requires the ability to provide a `onContextMenu`  propagated through the `DataDecorator` which always provides an `onClick`.